var/spack/repos/builtin/packages/libtiff/package.py

```diff
@@ -55,6 +55,18 @@ class Libtiff(AutotoolsPackage):
     conflicts('+zstd', when='@:4.0.9')
     conflicts('+webp', when='@:4.0.9')
 
+    # https://gitlab.com/libtiff/libtiff/-/merge_requests/243
+    patch('no-include-root.patch', when='@4.3.0')
```

Comment:
Maybe link the patch directly
```
https://gitlab.com/libtiff/libtiff/-/commit/b25618f6fcaf5b39f0a5b6be3ab2fb288cf7a75b.diff
```
?

Reply:
This patch has been modified from that patch. I think it's better to link to the PR since it provides a lot of context as to when and why the patch is actually needed, not just the patch itself. The PR links to the patch, but not the other way around.